google-cloud-storage/src/main/java/com/google/cloud/storage/ObjectLockMode.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.storage;
+
+import com.google.api.core.ApiFunction;
+import com.google.cloud.StringEnumType;
+import com.google.cloud.StringEnumValue;
+
+/**
+ * Represents the object lock mode. See <a
+ * href="https://cloud.google.com/storage/docs/object-lock">https://cloud.google.com/storage/docs/object-lock</a>
+ * for details.
+ */
+public final class ObjectLockMode extends StringEnumValue {
```

Comment:
This should probably move into the the `com.google.cloud.storage.multipartupload.model` package.

Reply:
ObjectLockMode is not specific to MPU but a general object property. Refer: https://cloud.google.com/storage/docs/object-lock.

IMO it should stay out of MPU specific model.

Reply:
In that case, sure I'll move it.